external-crates/move/crates/move-vm-config/src/verifier.rs

```diff
@@ -63,6 +64,7 @@ impl Default for VerifierConfig {
             max_constant_vector_len: Some(DEFAULT_MAX_CONSTANT_VECTOR_LEN),
             max_idenfitier_len: Some(DEFAULT_MAX_IDENTIFIER_LENGTH),
             allow_receiving_object_id: true,
+            reject_mutable_random_on_entry_functions: false,
```

Comment:
should the default here be true like for `allow_receiving_object_id`?

Reply:
I'm not sure exactly where the default is used. @tzakian, I'm assuming you added the one above

Reply:
It's used in a bunch of places for local development/tests. I would IMO default to whatever the most conservative setting here is/what will be used "in prod" (which from the sounds of it would be `true`?)